6.6/scripts/solr-foreground

```diff
@@ -16,4 +16,4 @@ fi
 
 echo "Starting Solr $SOLR_VERSION"
 
-exec solr -f "$@"
+exec solr -f -a "-XX:OnOutOfMemoryError=/opt/docker-solr/scripts/oom_solr.sh" "$@"
```

Comment:
Yea, hmm, that's not ideal. Let me think about that a bit more.

Reply:
Yeah, this is a nasty one.
[`$SOLR_ADDL_ARGS`](https://github.com/apache/lucene-solr/blob/master/solr/bin/solr#L2124) seems to be exactly there for stuff like this but I couldn't get it working. I always struggled with the escaping of the path to the `oom_solr.sh` script. 
If we manage to use `$SOLR_ADDL_ARGS` we probably could transparently adjust it to contain the OOM handling if not defined.